src/theme/litmus-portal.ts

```diff
@@ -20,30 +20,30 @@ const litmusPortalTheme = createTheme({
       toolTip: "#5252F6",
       legendTableHeading: "#0098DD",
       radialChartPassed: "#0098DD",
-      line: {
-        violet: "#A93DDB",
-        violetLight: "#A05195",
-        brightPink: "#D45087",
-        darkPink: "#CC556A",
-        yellow: "#FFA600",
-        darkYellow: "#DFA73E",
-        orange: "#F6793E",
-        lightGreen: "#6DA966",
-        lightBlue: "#51C9DA",
-        darkBlue: "#2F4B7C",
-      },
-      area: {
-        violet: "#A93DDB73",
-        violetLight: "#A0519573",
-        brightPink: "#A4508773",
-        darkPink: "#CC556A73",
-        yellow: " #FFA60073",
-        darkYellow: "#DFA73E73",
-        orange: "#F6793E73",
-        lightGreen: "#6DA96673",
-        lightBlue: "#51C9DA73",
-        darkBlue: "#2F4B7C73",
-      },
+      line: [
+        "#A93DDB",
+        "#A05195",
+        "#D45087",
+        "#CC556A",
+        "#FFA600",
+        "#DFA73E",
+        "#F6793E",
+        "#6DA966",
+        "#51C9DA",
+        "#2F4B7C",
+      ],
+      area: [
+        "#A93DDB73",
+        "#A0519573",
+        "#A4508773",
+        "#CC556A73",
+        " #FFA60073",
+        "#DFA73E73",
+        "#F6793E73",
+        "#6DA96673",
+        "#51C9DA73",
+        "#2F4B7C73",
+      ],
```

Comment:
I feel like this is not really readable to outside contributors, writing `graph.line.purple` makes more sense than `graph.line[3]` or something similar. When it comes to theme augmentation we can just make graph an optional parameter to not make it mandatory to fill these colors in the extended project.

Reply:
as per current configurations in Litmus. The user is not given an option to select the color for each metric as that number of metric in a panel is unknow by the user also. So all the metrics are assigned colors with map function using index. So having colors in array is better than as object